wallet/utxos.go

```diff
@@ -135,7 +135,7 @@ func (w *Wallet) FetchInputInfo(prevOut *wire.OutPoint) (*wire.MsgTx,
 	}
 	pubKeyAddr, ok := addr.(waddrmgr.ManagedPubKeyAddress)
 	if !ok {
-		return nil, nil, nil, 0, err
+		return nil, nil, nil, 0, ErrNotMine
```

Comment:
So do we need to modify this routine to support the other types? Or is this just a way to catch this case in some higher level code?

Reply:
Ah also weird interaction re the `err` variable shadowing 🙈

Reply:
I don't think we need to handle the other types, really. This is mainly about returning the _wrong_ error (it's always `nil` which is what breaks things at the call sites).